hubblestack_nova_profiles/cis/amazon-level-1-scored-v1-0-0.yaml

```diff
@@ -0,0 +1,814 @@
+            uid: 0
+            group: 'root'
+            gid: 0
+    description: '/etc/crontab must be owned by root and have persmissions 600'
+
+  cron_hourly:
+    data:
+      'Amazon Linux*':
+        - '/etc/cron.hourly':
+            tag: 'CIS-6.1.5'
+            mode: 700
+            user: 'root'
+            uid: 0
+            group: 'root'
+            gid: 0
+    description: '/etc/cron.hourly must be owned by root and must have permissions 700'
+
+  cron_daily:
+    data:
+      'Amazon Linux*':
+        - '/etc/cron.daily':
+            tag: 'CIS-6.1.6'
+            mode: 700
+            user: 'root'
+            uid: 0
+            group: 'root'
+            gid: 0
+    description: '/etc/cron.daily must be owned by root and must have permissions 700'
+
+  cron_weekly:
+    data:
+      'Amazon Linux*':
+        - '/etc/cron.weekly':
+            tag: 'CIS-6.1.7'
+            mode: 700
+            user: 'root'
+            uid: 0
+            group: 'root'
+            gid: 0
+    description: '/etc/cron.weekly must be owned by root and must have permissions 700'
+
+  cron_monthly:
+    data:
+      'Amazon Linux*':
+        - '/etc/cron.monthly':
+            tag: 'CIS-6.1.8'
+            mode: 700
+            user: 'root'
+            uid: 0
+            group: 'root'
+            gid: 0
+    description: '/etc/cron.monthly must be owned by root and must have permissions 700'
+
+  cron_d:
+    data:
+      'Amazon Linux*':
+        - '/etc/cron.d':
+            tag: 'CIS-6.1.9'
+            mode: 700
+            user: 'root'
+            uid: 0
+            group: 'root'
+            gid: 0
+    description: '/etc/cron.d must be owned by root and must have permissions 700'
+
+  at_allow:
+    data:
+      'Amazon Linux*':
+        - '/etc/at.allow':
+            tag: 'CIS-6.1.10'
+            mode: 600
+            user: 'root'
+            uid: 0
+            group: 'root'
+            gid: 0
+    description: '/etc/at.allow must be owned by root and have persmissions 600'
+
+  at_cron_allow:
+    data:
+      'Amazon Linux*':
+        - '/etc/cron.deny':
+            tag: 'CIS-6.1.11'
+            mode: null
+            user: null
+            uid: null
+            group: null
+            gid: null
+        - '/etc/at.deny':
+            tag: 'CIS-6.1.11'
+            mode: null
+            user: null
+            uid: null
+            group: null
+            gid: null
+        - '/etc/cron.allow':
+            tag: 'CIS-6.1.11'
+            mode: 600
+            user: 'root'
+            uid: 0
+            group: 'root'
+            gid: 0
+        - '/etc/at.allow':
+            tag: 'CIS-6.1.11'
+            mode: 600
+            user: 'root'
+            uid: 0
+            group: 'root'
+            gid: 0
+    description: 'Restrict at/cron to authorized users'
+
+  sshd_config:
+    data:
+      'Amazon Linux*':
+        - '/etc/ssh/sshd_config':
+            tag: 'CIS-6.2.3'
+            mode: 600
```

Comment:
As per CIS doc, 'The /etc/ssh/sshd_config file needs to be protected from unauthorized changes by non- privileged users, but needs to be readable as this information is used with many non- privileged programs.' so this mean 644 should also be fine. We need to add OR check here.